internal/clients/template.go

```diff
@@ -86,15 +74,19 @@ func TerraformSetupBuilder(version, providerSource, providerVersion string) terr
 			return ps, errors.Wrap(err, errUnmarshalCredentials)
 		}
 
-		// set provider configuration
-		ps.Configuration = map[string]interface{}{
-			"host": templateCreds[keyHost],
-		}
 		// set environment variables for sensitive provider configuration
-		ps.Env = []string{
-			fmt.Sprintf(fmtEnvVar, envUsername, templateCreds[keyUsername]),
-			fmt.Sprintf(fmtEnvVar, envPassword, templateCreds[keyPassword]),
-		}
+		// Deprecated: In shared gRPC mode we do not support injecting
+		// credentials via the environment variables. You should specify
+		// credentials via the Terraform main.tf.json instead.
+		/*ps.Env = []string{
+			fmt.Sprintf("%s=%s", "HASHICUPS_USERNAME", templateCreds["username"]),
+			fmt.Sprintf("%s=%s", "HASHICUPS_PASSWORD", templateCreds["password"]),
+		}*/
+		// set credentials in Terraform provider configuration
+		/*ps.Configuration = map[string]interface{}{
+			"username": templateCreds["username"],
+			"password": templateCreds["password"],
+		}*/
```

Comment:
```suggestion
		ps.Configuration = map[string]interface{}{
			"username": templateCreds["username"],
			"password": templateCreds["password"],
		}
```

I think it's fine to have this uncommented to keep in working state by default.

Reply:
[null provider](https://registry.terraform.io/providers/hashicorp/null/latest/docs) does not have any configuration options.